bundles/binding/org.openhab.binding.akm868/src/main/java/org/openhab/binding/akm868/internal/AKM868Binding.java

```diff
@@ -0,0 +1,205 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.akm868.internal;
+
+import java.util.Dictionary;
+
+import java.util.List;
+
+import org.openhab.binding.akm868.AKM868BindingProvider;
+
+import org.osgi.service.cm.ConfigurationException;
+import org.osgi.service.cm.ManagedService;
+import org.apache.commons.lang.StringUtils;
+import org.openhab.core.binding.AbstractActiveBinding;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+	
+
+/**
+ * Implement this class if you are going create an actively polling service
+ * like querying a Website/Device.
+ * 
+ * @author Michael Heckmann
+ * @since 1.7.0
+ */
+public class AKM868Binding extends AbstractActiveBinding<AKM868BindingProvider> implements ManagedService, AKM868Listener {
+
+	private static final Logger logger = 
+		LoggerFactory.getLogger(AKM868Binding.class);
+
+	
+	/** 
+	 * the refresh interval which is used to poll values from the AKM868
+	 * server (optional, defaults to 60000ms)
+	 */
+	private long refreshInterval = 60000;
+	private String host;
+	private int port;
+	private long timeout;
+	private AKM868PacketReceiver packetlistener;
+	
+	
+	public AKM868Binding() {
+		packetlistener = new AKM868PacketReceiver(this);
+	}
+		
+	
+	public void activate() {
+		
+	}
+	
+	public void deactivate() {
+		logger.debug("Stoppig AKM868 listener...");
```

Comment:
i would expect a "stop listener" message in a method stopListener(). Either move the statement or change the wording